src/Widgets/DividerWidget/DividerWidgetComponent.js

```diff
@@ -9,7 +9,7 @@ Scrivito.provideComponent("DividerWidget", ({ widget }) => {
   if (widget.get("showLogo") && root) {
     return (
       <div className="divider-widget">
-        <Scrivito.ImageTag content={root.get("dividerLogo")} />
+        <Scrivito.ImageTag content={root.get("dividerLogo")} alt="Logo" />
```

Comment:
* [ ] Please see https://accessibleweb.com/question-answer/when-should-i-use-a-null-or-empty-alt-tag/
> Decorative images do not need to be announced by the screen reader, so if the alt attribute is empty (alt="", aka a "null" tag) it will not be announced to the user